README.md

```diff
@@ -115,6 +115,65 @@ export default function Home() {
 </script>
 ```
 
+## Player Preset Variants
+
+ThorVG Lottie Player provides multiple presets optimized for different use cases. Each preset can be selected based on bundle size and performance requirements.
+
+### Standard Presets
+- **Software Renderer (SW)**: CPU-based rendering with all Lottie spec supported
+- **WebGL Renderer (GL)**: GPU-accelerated rendering with all Lottie spec supported
+
+### Lite Presets
+- **Software Lite (SW-Lite)**: CPU-based rendering with minimal Lottie spec (PNG only, without Expressions)
```

Comment:
`Software Lite (SW-Lite): A CPU-based renderer that supports basic Lottie specification (PNG only; Fonts and Expressions are not supported).`